cmd/convox/rack.go

```diff
@@ -205,6 +205,27 @@ func cmdRackParamsSet(c *cli.Context) error {
 		params[parts[0]] = parts[1]
 	}
 
+	existing, err := rackClient(c).ListParameters(system.Name)
+	if err != nil {
+		return stdcli.Error(err)
+	}
+
+	hasChange := false
+	for k, v := range params {
+		res, ok := existing[k]
+		if !ok {
+			hasChange = true
+			break
+		} else if res != v {
+			hasChange = true
+			break
+		}
+	}
+
+	if !hasChange {
```

Comment:
AWS will tell us if it won't make a change. Let's look for that error message and leave the logic here up to the CF engine.